WalletWasabi.Fluent/Helpers/StartupHelper.cs

```diff
@@ -62,5 +62,33 @@ private static void StartOnWindowsStartup(bool runOnSystemStartup, string pathTo
 				key.DeleteValue(nameof(WalletWasabi), false);
 			}
 		}
+
+		private static async Task StartOnLinuxStartupAsync(bool runOnSystemStartup)
+		{
+			string pathToDir = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.UserProfile), ".config", "autostart");
+			string pathToDesktopFile = Path.Combine(pathToDir, "Wasabi.desktop");
+
+			IoHelpers.EnsureContainingDirectoryExists(pathToDesktopFile);
+
+			if (runOnSystemStartup)
+			{
+				string pathToExec = EnvironmentHelpers.GetExecutablePath();
+				string fileContent = string.Join(
+					"\n",
+					"[Desktop Entry]",
+					"Name=WasabiWallet",
```

Comment:
I think you have introduced a constant for this recently.

Reply:
You are right 😅